lua/plenary/path.lua

```diff
@@ -166,7 +166,7 @@ function Path:absolute()
   if self:is_absolute() then
     return self.filename
   else
-    return self._absolute or table.concat({self._cwd, self.filename}, self.sep)
+    return self._absolute or table.concat({self._cwd, self.filename}, self._sep)
```

Comment:
Its `self._sep` not `self.sep` that actually meant we created files like that:
`cwdname.lua` rather then `cwd/name.lua`